db/migrate/20160324174132_create_container_deployments.rb

```diff
@@ -0,0 +1,22 @@
+class CreateContainerDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string     :deployment_type
+      t.string     :deployment_version
+      t.boolean    :containerized
+      t.string     :deployment_method
```

Comment:
Are the `deployment_` prefixes redundant considering a record in this table would be called a ContainerDeployment?  i.e. it would read like `container_deployment.deployment_method`

If you dropped the `deployment_` prefix, then version is fine, but of course type and method might be problematic...perhaps that's a good thing as maybe those names are too generic?